server/create_tables.js

```diff
@@ -11,7 +11,7 @@ var myCon = mysql.createConnection({
 });
 
 // Drops database to be created again after change to SQL schema
-// myCon.query("DROP DATABASE LIHI;", function (err, result) {
+// myCon.query("DROP DATABASE IF EXISTS LIHI;", function (err, result) {
 // 	if (err) throw err;
 // 	console.log("Database deleted");
 // });
```

Comment:
If this is commented out I am guessing this was a one time code and we can remove it.

Reply:
This code can be uncommented when resetting the database is necessary. It was found useful for testing but isn't needed for production.

Reply:
Agreed
Do you believe we would still need this?
If so, please add an issue that reminds us to remove this or a TODO in the code itself

Reply:
I believe it'll still be useful until we finalize deployment. I've created an issue for it.